src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/Checker.java

```diff
@@ -263,6 +263,13 @@ public Void scan(DocCommentTree tree, TreePath p) {
         return null;
     }
 
+    // Checks if the passed tree path does NOT correspond to an entity, such as
+    // the overview file and doc-files/**/*.html files.
+    private boolean notPseudoElement(TreePath p) {
```

Comment:
The use of a predicate named `not...` seems confusing and leads to various double negatives;  it would be easier to read if this was `isPseudoElement` even if that means an extra `!` at the call site.

Reply:
> It worries me that we are removing support for a feature that had explicit positive tests.

We discussed this offline at length.   The tests are not specifically for the feature, but instead, they are tests for a different feature (relative links) in a variety of situations, including the suspect case of `{@inheritDoc`.

It might be worth investigating whether we can write tests for relative links when `{@inheritDoc}` is used in a legal position, such as on a method declaration.